src/views/BusinessCase/Review/ProposedBusinessCaseSolutionReview.tsx

```diff
@@ -31,100 +31,103 @@ type ReviewProps = {
-  <DescriptionList title={name}>
-    <ReviewRow>
-      <div className="line-height-body-3">
-        <DescriptionTerm term={`${name}: Title`} />
-        <DescriptionDefinition definition={solution.title} />
-      </div>
-    </ReviewRow>
-    <ReviewRow>
-      <div className="line-height-body-3">
-        <DescriptionTerm term={`${name}: Summary`} />
-        <DescriptionDefinition
-          className="text-pre-wrap"
-          definition={solution.summary}
-        />
-      </div>
-    </ReviewRow>
-    <ReviewRow>
-      <div className="line-height-body-3">
-        <DescriptionTerm term={`${name}: Acquisition approach`} />
-        <DescriptionDefinition
-          className="text-pre-wrap"
-          definition={solution.acquisitionApproach}
-        />
-      </div>
-    </ReviewRow>
-    <ReviewRow>
-      <div className="line-height-body-3">
-        <DescriptionTerm term="Do you need to host your solution?" />
-        <DescriptionDefinition
-          className="text-pre-wrap"
-          definition={hostingTypeMap[solution.hosting.type]}
-        />
-      </div>
-    </ReviewRow>
-    {['cloud', 'dataCenter'].includes(solution.hosting.type) && (
+  <>
+    <h3 className="easi-only-print business-case-solution-header">{name}</h3>
+    <DescriptionList title={name}>
       <ReviewRow>
         <div className="line-height-body-3">
-          <DescriptionTerm term="Where are you planning to host?" />
+          <DescriptionTerm term={`${name}: Title`} />
+          <DescriptionDefinition definition={solution.title} />
+        </div>
+      </ReviewRow>
+      <ReviewRow>
+        <div className="line-height-body-3">
+          <DescriptionTerm term={`${name}: Summary`} />
+          <DescriptionDefinition
+            className="text-pre-wrap"
+            definition={solution.summary}
+          />
+        </div>
+      </ReviewRow>
+      <ReviewRow>
+        <div className="line-height-body-3">
+          <DescriptionTerm term={`${name}: Acquisition approach`} />
+          <DescriptionDefinition
+            className="text-pre-wrap"
+            definition={solution.acquisitionApproach}
+          />
+        </div>
+      </ReviewRow>
+      <ReviewRow>
+        <div className="line-height-body-3">
+          <DescriptionTerm term="Do you need to host your solution?" />
           <DescriptionDefinition
             className="text-pre-wrap"
-            definition={solution.hosting.location}
+            definition={hostingTypeMap[solution.hosting.type]}
           />
         </div>
-        {solution.hosting.cloudServiceType && (
+      </ReviewRow>
+      {['cloud', 'dataCenter'].includes(solution.hosting.type) && (
+        <ReviewRow>
           <div className="line-height-body-3">
-            <DescriptionTerm term="What, if any, type of cloud service are you planning to use for this solution (Iaas, PaaS, SaaS, etc.)?" />
+            <DescriptionTerm term="Where are you planning to host?" />
```

Comment:
I think git is just doing a strange diff thing here- I didn't change the guts of this component.